tests/test_verifylib.py

```diff
@@ -29,6 +29,7 @@
 import shutil
 import tempfile
 import unittest
+import pytz
```

Comment:
Instead of using another dependency I suggest using `datetime.timezone.utc` or `dateutil.tz.tzutc`. The latter is also a third-party dependency but we already use it for `relativedelta`.

Reply:
Also the CI's likely to fail as this isn't installed.